packages/bpk-component-info-banner/src/BpkInfoBanner.module.scss

```
@@ -0,0 +1,115 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../bpk-mixins/index.scss';
+
+.bpk-info-banner {
+  padding: bpk-spacing-md() bpk-spacing-base();
+  overflow: hidden;
+
+  @include info-banner;
+
+  &--style-default {
+    background-color: $bpk-canvas-contrast-day;
+  }
+
+  &--style-onContrast {
+    background-color: $bpk-surface-default-day;
+  }
+
+  &__header {
+    display: flex;
+
+    // padding: bpk-spacing-md() bpk-spacing-base();
+    flex-direction: row;
+    align-items: flex-start;
+
+    &--expandable {
+      cursor: pointer;
+    }
+  }
+
+  &__icon,
+  &__toggle {
+    padding: bpk-spacing-sm() / 2 0;
+  }
+
+  // Make sure the items in the flexbox are display: inline-block; for older IE's
+  &__icon,
+  &__message,
+  &__toggle {
+    display: inline-block;
+  }
+
+  &__message {
+    padding: bpk-spacing-sm() bpk-spacing-md();
+    flex-grow: 1;
+  }
+
+  &__success-icon {
+    @include bpk-themeable-property(
+      fill,
+      --bpk-private-info-banner-success-color,
+      $bpk-private-info-banner-success-color
+    );
```

We can probably ignore the requirement of needing to theme components anymore and just set it to the colour it should be

```suggestion
    fill: $bpk-private-info-banner-success-color;
```